pkg/serviceaccount/jwt.go

```diff
@@ -313,8 +313,9 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(ctx context.Context, tokenData
 	}
 
 	return &authenticator.Response{
-		User:      sa.UserInfo(),
-		Audiences: auds,
+		User:              sa.UserInfo(),
```

Comment:
What are the expected values here? I'm concerned that this will cause issues, since it seems from the face of it, to be completely unbounded.

Reply:
Hmm.. good point maybe this should be an enum.